README.md

```diff
@@ -135,77 +101,23 @@ setup, which includes the [scheduler and monitoring services][docker-compose-sch
 [learning-resources]: https://docs.hyperswitch.io/learn-more/payment-flows
 [local-setup-guide]: /docs/try_local_system.md
 [docker-compose-scheduler-monitoring]: /docs/try_local_system.md#running-additional-services
-<a href="#Fast-Integration-for-Stripe-Users">
-  <h2 id="fast-integration-for-stripe-users">🔌 Fast Integration for Stripe Users</h2>
-</a>
-
-If you are already using Stripe, integrating with Hyperswitch is fun, fast & easy.
-Try the steps below to get a feel for how quick the setup is:
 
-1. Get API keys from our [dashboard].
-2. Follow the instructions detailed on our
-   [documentation page][migrate-from-stripe].
-
-[dashboard]: https://app.hyperswitch.io/register
-[migrate-from-stripe]: https://hyperswitch.io/docs/migrateFromStripe
-
-<a href="#Supported-Features">
-  <h2 id="supported-features">✅ Supported Features</h2>
+<a href="https://app.hyperswitch.io/">
```

Comment:
Hey @neerajkumar-n Can we please move this section before the Quick Setup Section.

Reply:
Don't think that would make sense, the updated format focuses more on Contribution optimization. There is again a TOC - from which any Contributor can Navigate to Trying a Payment.

This is experimental content, which I will be updating over the next few weeks to make it optimal